### PR TITLE
Nunu-0 1 Week 

### PR DESCRIPTION
**Pull request by Nunu-0**
test
test
test

### CODE REVIEW

File: week1/test.cpp [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include <iostream>
+using namespace std;
```
Comment (Nunu-0) [inline week1/test.cpp:2]: d

File: week1/test.cpp [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+int main(){
```
Comment (Nunu-0) [inline week1/test.cpp:3]: d

**Review by Nunu-0 — COMMENTED** [review]
test

File: week1/test.cpp [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#include <iostream>
```
Comment (minji-o-j) [inline week1/test.cpp:1]: gogo

File: week1/test.cpp [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+using namespace std;
+int main(){
+  cout >> "Hello";
```
Comment (minji-o-j) [inline week1/test.cpp:4]: - 

- 

-

**Pull request merged**